components/bulletins/BulletinCard.vue

```diff
@@ -84,29 +88,25 @@ export default {
 
 .bulletinCard__content > h2 {
     @apply text-base text-center mt-6 mb-5 font-bold;
-    @media (min-width: 415px) {
+    font-family: 'Noto Serif TC';
+    @media (min-width: 375px) {
         @apply text-xl mt-8 mb-5;
     }
-    color: #808080;
-}
-
-.bulletinCard__content:hover > h2 {
-    color: #7568f6;
 }
 
 .bulletinCard__content > p {
     @apply mx-2;
     font-size: 0.6rem;
-    color: #c7c7c7;
-    @media (min-width: 415px) {
+    font-family: 'Source Sans Pro';
+    @media (min-width: 375px) {
         font-size: 0.8rem;
     }
 }
 
-.bulletinCard__button {
+/* .bulletinCard__button {
     @apply absolute bottom-5 cursor-pointer;
     @media (min-width: 415px) {
         @apply bottom-7;
     }
-}
+} */
```

Comment:
Same as L43 - L45. We could keep it If it will be used in the future.

Reply:
Uncommented.